.github/workflows/build.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
     name: Run Rustfmt
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@01aecccf739ca6ff86c0539fbc67a7a5007bbc81 # renovate: tag=v2.1.0
```

Comment:
Hi @soenkeliebau, I saw your changes your made here https://github.com/renovatebot/renovate/pull/14110 and I am asking myself if it will be worth my time to create a new PR which also handles ratchets pinning comments?

https://github.com/sethvargo/ratchet

```
uses: 'actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b' # ratchet:actions/checkout@v3
# or
image: 'ubuntu@sha256:47f14534bda344d9fe6ffd6effb95eefe579f4be0d508b7445cf77f61a0e5724' # ratchet:ubuntu:20.04
```